src/logistration/tests/RegistrationPage.test.jsx

```diff
@@ -57,13 +265,40 @@ describe('./RegistrationPage.js', () => {
   it('should match url after redirection', () => {
     const dasboardUrl = 'http://test.com/testing-dashboard/';
     store = mockStore({
-      ...store,
+      ...initialState,
       logistration: {
-        ...store.logistration,
+        ...initialState.logistration,
         registrationResult: {
+          ...initialState.logistration.registrationResult,
```

Comment:
nit: we don't need it since we are updating the `registrationResult` completely

Reply:
done